### ISSUE TITLE
双通道注意力融合的这两段代码能解释一下吗？

### ISSUE THREAD

**Issue by cqlouis** [lmy1997/tuxiaoshi#3]
attention_weights = F.softmax(torch.randn(x1.size(0), 1), dim=0).cuda()
x = attention_weights * x1 + (1 - attention_weights) * x2

这两行代码没看懂，尤其是第一行，这种注意力怎么来的？

Thanks！
